Algorithms/Stack/InfixToPostfix.java

```diff
@@ -0,0 +1,70 @@
+
+import java.util.*;
+import java.lang.*;
+import java.io.*;
+
```

Comment:
the name, purpose, sample input-outputs mention it in the comment section

Reply:
This isnt solved

Reply:
I have searched everywhere and collapsed that import statement at its extent. Now it can't be further collapsed.

Reply:
Made changes as per your request/

Reply:
Searched everywhere and compressed the import statement at it's extent. Now it can't be further compressed. Please have a look.

Reply:
> the name, purpose, sample input-outputs mention it in the comment section

I meant this

Reply:
Searched everywhere and compressed the import statement to it's extent. It can't be further compressed. Please have a look.

Reply:
I am asking you to write down the algorithm name and purpose in the comments
Get it?

Reply:
I have commented sample input-output above the code by making comments.
Please correct if i am not getting you.

Reply:
Comments as in, in the code not here